### PR TITLE
zh: docs/tasks/traffic-management/secure-ingress/sds/index.md

### PR DESCRIPTION
**Pull request by fleeto**

### CODE REVIEW

File: content_zh/docs/tasks/traffic-management/secure-ingress/sds/index.md [hunk 1]
```diff
@@ -0,0 +1,540 @@
+---
+title: 使用 SDS为 Gateway 提供 HTTPS 加密支持
```
[the remark this answers is not in the document]
Comment (SataQiu) [reply content_zh/docs/tasks/traffic-management/secure-ingress/sds/index.md]: 一个小建议！

File: content_zh/docs/tasks/traffic-management/secure-ingress/sds/index.md [hunk 2]
```diff
@@ -0,0 +1,540 @@
...
+          containers:
+          - image: docker.io/citizenstig/httpbin
+            imagePullPolicy: IfNotPresent
+            name: httpbin
+            ports:
+            - containerPort: 8000
+    EOF
+    {{< /text >}}
+
+1. 为 Ingress 网关创建 `Secret：`
```
Comment (Mr-Linus) [inline content_zh/docs/tasks/traffic-management/secure-ingress/sds/index.md]:
```suggestion
1. 为 Ingress 网关创建 `Secret` ：
```

File: content_zh/docs/tasks/traffic-management/secure-ingress/sds/index.md [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+          protocol: HTTPS
+        tls:
+          mode: SIMPLE
+          credentialName: "httpbin-credential" # 和 Secret 名称一致
+        hosts:
+        - "httpbin.example.com"
+    EOF
+    {{< /text >}}
+
+1. 配置网关的 Ingress 流量路由，并配置对应的 `VirtualService`：：
```
Comment (Mr-Linus) [inline content_zh/docs/tasks/traffic-management/secure-ingress/sds/index.md]:
```suggestion
1. 配置网关的 Ingress 流量路由，并配置对应的 `VirtualService`：
```

File: content_zh/docs/tasks/traffic-management/secure-ingress/sds/index.md [hunk 4]
```diff
@@ -0,0 +1,540 @@
...
+          protocol: HTTPS
+        tls:
+          mode: SIMPLE
+          credentialName: "helloworld-credential"
+        hosts:
+        - "helloworld-v1.example.com"
+    EOF
+    {{< /text >}}
+
+1. 配置网关的流量路由，配置 `VirtualService`：、
```
Comment (Mr-Linus) [inline content_zh/docs/tasks/traffic-management/secure-ingress/sds/index.md]:
```suggestion
1. 配置网关的流量路由，配置 `VirtualService`：
```

**Pull request merged**